### ISSUE TITLE
Mouse scrolling not working properly

### ISSUE THREAD

**Issue by lunemec** [jonathanslenders/pyvim#40]
In vim when you want mouse scrolling to scroll instead of just moving your cursor, you set in your .vimrc these:
```
set mouse=a
set ttymouse=xterm2
```
I tried to set these options in pyvim, but there are none. They should either be options, or enabled by default.

**Comment by jonathanslenders** [comment]
Yes, we don't have mouse support yet... I really hope to be able to implement that some day. That would be great for all applications using prompt-toolkit. But it's not an easy one. So, probably I will postpone this until all other important things are working.

**Comment by dragon788** [comment]
Mouse support appears to be working for me in PyVim even under PyMux.
